contrib/endpoints/src/api_manager/firebase_rules/firebase_request.h

```diff
@@ -0,0 +1,155 @@
+// Copyright 2017 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+////////////////////////////////////////////////////////////////////////////////
+
+#ifndef FIREBASE_RULES_FIREBASE_REQUEST_H_
+#define FIREBASE_RULES_FIREBASE_REQUEST_H_
+
+#include <string>
+#include <utility>
+#include <vector>
+#include "contrib/endpoints/include/api_manager/utils/status.h"
+#include "contrib/endpoints/src/api_manager/context/request_context.h"
+#include "contrib/endpoints/src/api_manager/proto/security_rules.pb.h"
+
+using ::google::api_manager::utils::Status;
+using TestRulesetResponse = ::google::api_manager::proto::TestRulesetResponse;
+using FunctionCall = TestRulesetResponse::TestResult::FunctionCall;
+using ::google::protobuf::RepeatedPtrField;
```

Comment:
per c++ style guide,  name space alias should not be used in the header file. It can only be used in the cc file. Otherwise, it will easily cause confusion.